server/api/src/rsGeneralAdmin.cpp

```diff
@@ -52,6 +53,59 @@ namespace
         return std::find_if(b, e, [](unsigned char _ch) { return ::isspace(_ch); }) != e;
     } // contains_whitespace
 
+     auto throw_if_user_manages_an_irods_server(
+        RsComm& rsComm,
+        const std::string_view _new_user_type,
+        const std::string_view _user_name
```

Comment:
Let's swap the order of these parameters as well.

I mentioned this in a different comment, but we need to consider what the client is trying to change.

Therefore, this function should accept 4 parameters.
```cpp
RsComm& _comm,
const std::string_view _option,
const std::string_view _user_name,
const std::string_view _new_user_type
```
Then, if `_option` is not equal to `"type"`, we return early.